hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CreateTableProcedure.java

```
@@ -292,6 +297,24 @@ private boolean prepareCreate(final MasterProcedureEnv env) throws IOException {
       return false;
     }
 
+    ColumnFamilyDescriptor[] columnFamilyDescriptors = tableDescriptor.getColumnFamilies();
```

You can add a check ti fail fast at https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/util/TableDescriptorChecker.java#L213

---

Thanks for reviewing, The updates have been made.